packages/desktop-client/src/components/accounts/Account.tsx

```diff
@@ -146,7 +150,7 @@ type AllTransactionsProps = {
   collapseTransactions: (ids: string[]) => void;
 };
 
-function AllTransactions({
+export function AllTransactions({
```

Comment:
Warning: this component is about the be removed in one of the in progress PRs: https://github.com/actualbudget/actual/pull/3708/files#diff-99c276500045bfda8f9c14b3c10705730a1fe7452a69246d4e40d092c24d305eL149

Reply:
See https://github.com/actualbudget/actual/pull/3708/files#diff-99c276500045bfda8f9c14b3c10705730a1fe7452a69246d4e40d092c24d305eR508 instead.

Reply:
oh, I added this today cuz to fix the missing split transactions.
can't find the changes you referenced.
whats the name of the file so I can filter it?

Reply:
got my way in

Reply:
fixed

Reply:
btw, I will have to go back to this transaction list to make it more functional